solr/solr-ref-guide/modules/deployment-guide/pages/user-managed-index-replication.adoc

```diff
@@ -378,9 +378,26 @@ http://_leader_host:port_/solr/_core_name_/replication?command=disablereplicatio
 `indexversion`::
 Return the version of the latest replicatable index on the specified leader or follower.
 +
+====
+[.tab-label]*V1 API*
+
 [source,bash]
+----
 http://_host:port_/solr/_core_name_/replication?command=indexversion
 
+----
+====
++
+====
+[.tab-label]*V2 API*
+
+[source,bash]
+----
+http://_host:port_/api/_core_name_/replication/indexversion
```

Comment:
[-1] L396 should be `http://_host:port_/api/cores/_core_name_/replication/indexversion` I think. (The "/cores" segment is missing)

Reply:
Ah nice catch on the missed typo by me. Fixed